test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/RepositoryCountersignatureTests.cs

```diff
@@ -0,0 +1,400 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if NET46
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.Pkcs;
+using System.Security.Cryptography.X509Certificates;
+using NuGet.Common;
+using NuGet.Packaging.Signing;
+using NuGet.Packaging.Signing.DerEncoding;
+using Xunit;
+
+namespace NuGet.Packaging.Test
+{
+    public class RepositoryCountersignatureTests : IClassFixture<CertificatesFixture>
+    {
+        private readonly CertificatesFixture _fixture;
+
+        public RepositoryCountersignatureTests(CertificatesFixture fixture)
+        {
+            if (fixture == null)
+            {
+                throw new ArgumentNullException(nameof(fixture));
+            }
+
+            _fixture = fixture;
+        }
+
+        [Fact]
+        public void GetRepositoryCounterSignature_WithoutCountersignature_ReturnsNull()
```

Comment:
String replace in this file the case-sensitive string:

    GetRepositoryCounterSignature

...with:

    GetRepositoryCountersignature